app/views/layouts/_flashes.html.haml

```diff
@@ -1,19 +1,39 @@
 - flash.each do |key, value|
-  - case key
-    - when 'alert'
-      .govuk-error-summary
-        .govuk-error-summary__title
-          = value
-        / .govuk-error-summary__body
-        .govuk-error-message
+  -if value.is_a?(String)
```

Comment:
Could we add a space between the dash and `if` please. Same with line 20. I'm surprised the linter didn't pick this up

Reply:
Should be resolved with most recent push